frontend/src/actions/WindowActions.js

```diff
@@ -398,12 +398,15 @@ export function indicatorState(state) {
  */
 export function fetchTab({ tabId, windowId, docId, orderBy }) {
   return (dispatch) => {
+    const tableId = getTableId({ windowId, docId, tabId });
+    dispatch(updateTabTable({ tableId, tableResponse: {}, pending: true }));
```

Comment:
as described in the issue, don't clear the rows before getting new ones

Reply:
There's a messy logic in there with the creation of the table. I used this workaround to set the pending before. Seems it is not ok. Need to think for a solution. Thx

Reply:
@petrican  still not OK, and u know it. u don;t have to ask me the obvious :(
![image](https://user-images.githubusercontent.com/1244701/150492434-6e540604-1eae-41f6-8516-e00392b3c518.png)

Reply:
@petrican  what i would suggest to do here is to call the function without tableRespose param.
i.e.
```
updateTabTable({ tableId, pending: true })
```

and then change the updateTabTable or where is needed and if the `tableRespose` is missing, simply don't update/change the rows.

makes sense?

Reply:
Ok it make sense. Got it. But just to dbl check the final result should be as in this snapshot correct ?

<img width="1003" alt="Screenshot 2022-01-21 at 10 32 29" src="https://user-images.githubusercontent.com/1708561/150493573-80718fdb-83a4-4cef-98be-92cccd3de694.png">

Reply:
@petrican 
![image](https://user-images.githubusercontent.com/1244701/150494638-4177dec7-fdfa-4435-9cf3-a587b58d859a.png)


makes sense now?
any suggestions?

Reply:
ok. ty.

Reply:
Pushed the changes. This is how it works now https://www.loom.com/share/6007d237bea84505974c822fe706a02f

Reply:
<img width="839" alt="Screenshot 2022-01-24 at 09 07 35" src="https://user-images.githubusercontent.com/1708561/150737189-fa52bc18-0e08-45e4-89d8-783bd8ebb881.png">